### PR TITLE
Tyrus server support for native-image in SE

### LINKED ISSUE #1966: Tyrus support in GraalVM
## Environment Details
* Helidon Version: 2.0.0-RC1
* Helidon SE
* JDK version:GraalVM Version 20.1.0 (Java Version 11.0.7)
* OS: Ubuntu 18.04.4 LTS
* Docker version (if applicable):

----------

## Problem Description

I talked with @jsupol and he thinks we need to add the native properties in tyrus helidon module to allow reflection.

Exception in thread "main" java.lang.NullPointerException
	at org.glassfish.tyrus.core.TyrusEndpointWrapper.<init>(TyrusEndpointWrapper.java:250)
	at org.glassfish.tyrus.core.TyrusEndpointWrapper.<init>(TyrusEndpointWrapper.java:152)
	at org.glassfish.tyrus.core.TyrusWebSocketEngine.register(TyrusWebSocketEngine.java:639)
	at io.helidon.webserver.tyrus.TyrusSupport$Builder.lambda$build$1(TyrusSupport.java:198)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at io.helidon.webserver.tyrus.TyrusSupport$Builder.build(TyrusSupport.java:195)
	at io.helidon.webserver.examples.websocket.Main.createRouting(Main.java:54)
	at io.helidon.webserver.examples.websocket.Main.startWebServer(Main.java:59)
	at io.helidon.webserver.examples.websocket.Main.main(Main.java:85)
## Steps to reproduce
cd helidon/examples/webserver/websocket
mvn package -Pnative-image
./target/helidon-examples-webserver-websocket

### PR DESCRIPTION
**Pull request by danielkec**
This fixes Tyrus server, not the client

Fixes #1966

Signed-off-by: Daniel Kec <daniel.kec@oracle.com>

### CODE REVIEW

File: common/common/src/main/java/io/helidon/common/FeatureCatalog.java [hunk 1]
```diff
@@ -150,7 +150,7 @@
                     .name("Websocket")
                     .description("Jakarta Websocket implementation")
                     .path("WebServer", "Websocket")
-                    .nativeSupported(false));
+                    .nativeSupported(true));
```
Comment (tomas-langer) [inline common/common/src/main/java/io/helidon/common/FeatureCatalog.java:153]: Need to add nativeDescription that declares that only server is supported (unless this is feature for server only). If for server only, make sure we have a client feature as well that declares it is not supported (if possible).

Comment (danielkec) [reply common/common/src/main/java/io/helidon/common/FeatureCatalog.java]: fixed

**Comment by dansiviter** [comment]
Would `reflection-config.json` need changing for #1934?

**Comment by danielkec** [comment]
> Would `reflection-config.json` need changing for #1934?

@dansiviter Hard to say, I guess we will have to expand native image smoke test to find out

**Pull request merged**